### ISSUE TITLE
Sponsor: Azure Tech Community

### ISSUE THREAD

**Issue by ShMcK** [VanJS/meetup#34]
From an email I received from an Azure Tech Community Evangelist:

```
There are significant benefits in joining our Global Azure Tech Community which are outlined below.

As a verified Azure Tech Communities group, you’ll receive support, inspiration, and greater connection within the Global Azure Tech Community, including:
•	Microsoft will sponsor the monthly cost of a Meetup Pro subscription for your Meetup Group (valued at $35/month; the fee is paid directly to Meetup)
•	A badge on your Meetup’s landing page indicating active membership in the Azure Tech Community Network
•	Quarterly Meetup Organizer calls to connect with other Azure Evangelists and share best practices – you’ll build connections, share anecdotes, and learn from other great community leaders as the network grows
•	Coming soon, we’ll share Azure-related content available for use in future Meetup sessions

Participation in the Microsoft Azure Tech Communities network will be reviewed annually, with reviews starting March 5th, 2020. Participation is optional and can end any time at either your or Microsoft’s discretion. After participation ends billing will resume directly through Meetup at your chosen Meetup subscription level. 

Your Meetup’s participation in the Azure Tech Community Network further relies upon satisfying the following criteria:
•	A published Code of Conduct that is easily accessible from your Meetup group’s home page
•	You must maintain a regular meeting cadence of no less than 4 times per year

We’re counting on you to continue to run your Meetup as you always have – Meetup’s Terms and Conditions still apply. Microsoft’s role is simply to provide selected Meetup organizers with structure and support to help your group thrive and connect more Azure fans together in real life.

Ready to join the Azure Tech Communities?
```

TLDR

Benefits: Meetup Pro Account ($420/year) (https://www.meetup.com/pro)

Meetup Pro includes:
- improved user analytics
- improved communication through mailchimp integration (current plain text is a bit rough)
- ability to create networked mini-groups (VanJS workshops, etc)
- display sponsors, request sponsors, manage finances

There is a 30 day free trial for Meetup Pro available on the site.

Thoughts:
👍  or 👎

**Comment by ShMcK** [comment]
Sounds, I'll get back to them and see what we can do with meetup pro.
